src/new-client/src/Web/TearOutSection/handleTearOutSection.web.ts

```diff
@@ -0,0 +1,29 @@
+import { constructUrl } from "@/utils/url"
+import { openWindow } from "@/utils/window/openWindow"
+import { tearOutSection } from "./state"
+
+export function handleTearOutSection(section: string) {
+  let width = 380
+  let height = 380
+  if (section == "liverates") {
```

Comment:
An alternative would be add this config to an object and use section as the key, to remove if/else/else etc..

Example:
```
const sectionConfig = {
  liverates: {
    width: 1600,
    height: 800
  }
  ...
}

const { width, height } = sectionConfig[section]
```
and this can be enhanced with types once you add a `section` type

Reply:
Fixed, did it as you said.